domains/jariullah.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Personal Portfolio",
+  "repo": "https://github.com/Mohdjariullah/portfolio1",
+  "owner": {
+    "username": "mohdjariullah",
+    "email": "mohdjariullah@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://mohdjariullah.github.io/portfolio1/"
```

Comment:
CNAME can not contain `https://` or `/`

Reply:
factory.is-a.dev suggested this template to me with https:// and /

Reply:
![image](https://github.com/user-attachments/assets/a81ee84b-9900-4bc7-a0dc-6f2a1d30f624)
@Stef-00012